launch/tier4_control_launch/config/trajectory_follower/lateral_controller.param.yaml

```diff
@@ -44,8 +44,8 @@
     # -- vehicle model --
     vehicle_model_type: "kinematics" # vehicle model type for mpc prediction. option is kinematics, kinematics_no_delay, and dynamics
     input_delay: 0.24                # steering input delay time for delay compensation
-    vehicle_model_steer_tau: 0.1     # steering dynamics time constant (1d approximation) [s]
-    steer_lim_deg: 20.0              # steering angle limit [deg]
+    vehicle_model_steer_tau: 0.3     # steering dynamics time constant (1d approximation) [s]
```

Comment:
Is this change necessary to resolve the problem?
I'm wondering how big an impact this parameter has.

Reply:
@rej55 All parameters are set to those of autoware_launch.
https://github.com/tier4/autoware_launch/blob/tier4/universe/control_launch/config/trajectory_follower/mpc_follower.param.yaml

Reply:
OK, thank you!